src/Core/Entities/Cipher.cs

```diff
@@ -2,20 +2,22 @@
 using Bit.Core.Models.Data;
 using Bit.Core.Utilities;
 
+#nullable enable
+
 namespace Bit.Core.Entities;
 
 public class Cipher : ITableObject<Guid>, ICloneable
 {
-    private Dictionary<string, CipherAttachment.MetaData> _attachmentData;
+    private Dictionary<string, CipherAttachment.MetaData>? _attachmentData;
 
     public Guid Id { get; set; }
     public Guid? UserId { get; set; }
     public Guid? OrganizationId { get; set; }
     public Enums.CipherType Type { get; set; }
-    public string Data { get; set; }
-    public string Favorites { get; set; }
-    public string Folders { get; set; }
-    public string Attachments { get; set; }
+    public string Data { get; set; } = null!;
```

Comment:
Yep, but since we let Dapper, and EF do the initialization we just have to "promise" the compiler that we know what we are doing even though there isn't anything actually enforcing it. It unfortunately doesn't enforce us into setting this property if we create the object ourself but it will not let us set something that *is* nullable to it. When we can upgrade to C# 11 we can use a [`required`](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/required) property to alleviate all these issues.